src/System.Memory/src/System.Memory.csproj

```diff
@@ -14,8 +14,7 @@
     <IsPackable>true</IsPackable>
     <VersionPrefix>4.5.5</VersionPrefix>
     <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
-    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.1.2</AssemblyVersion>
-    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.2.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true'">4.0.2.0</AssemblyVersion>
```

Comment:
Confirmed that the inbox framework versions as `4.1.0.0`.
```
// C:\Users\ericstj\.nuget\packages\microsoft.netcore.app\2.1.0\ref\netcoreapp2.1\System.Memory.dll
// System.Memory, Version=4.1.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
```

I think we can remove the comment on line 8 about being frozen -- we're  saying we'll allow this to increment (so long as it stays below 4.1.x.x.